ea_largestcomponent.m

```diff
@@ -1,4 +1,15 @@
 function slicebw=ea_largestcomponent(slicebw)
+%
```

Comment:
@ningfei Please provide a description of the function, as well as the inputs and the outputs.